README.md

```diff
@@ -31,19 +31,48 @@ jobs:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
-By default, the app checks for readme in `readme.txt` and `plugin/readme.txt`. If the readme of your plugin is not in one of these locations, you can configure the app to look somewhere else by creating a file called `.wordpress-version-checker.json` in the root of your repo with the contents:
+## Configuration
+
+The app doesn't stricly require any configuration, however you can configure some aspects of its function by placing a file named `.wordpress-version-checker.json` in the root of your repository. The file may contain any of the following configuration options:
+
+### Plugin readme location.
+
+By default, the app checks for readme in `readme.txt` and `plugin/readme.txt`. If the readme of your plugin is not in one of these locations, you can configure the app to look somewhere else with the `readme` value in the configuration. The value can be either a single location or an array of locations to check - if multiple locations are provided, they will be checked in the given order until the first match.
+
+#### Examples
 
 ```json
 {
     "readme": "path/to/your/readme.txt"
 }
 ```
 
-If you want the issues to be automatically assigned to someone, you can put their GitHub usernames in the config as well:
+```json
+{
+    "readme": ["path/to/first/readme.txt", "path/to/second/readme.txt"]
+}
+```
+
+### WordPress release channel
+
+By default, the app will only notify you once the new WordPress version is released (**This will change** - starting from version 2.0, the default value will be changed to `rc`). By setting the `channel` value to one of `stable` or `rc`, you can choose to be notified when the new version is fully release, is in the release candidate (RC) stage of development or when the first beta versions are released.
```

Comment:
Seems as though `beta` is also an option for `channel`?

Reply:
I have temporarily disabled it in eaea4ffc4c17cf6c616aaf54c8786b7d0ad52ee9 - I have to wait until the betas start rolling again (late June I think) to implement the checks for them. RC should work as-is, though...

Reply:
Ok, seems good then, thanks!

Reply:
Great, thanks for the second pair of eyes :)